packages/Testing/PHPUnit/AbstractTestCase.php

```diff
@@ -72,12 +72,15 @@ protected function getService(string $type): object
      */
     private function createConfigsHash(array $configFiles): string
     {
-        Assert::allFile($configFiles);
```

Comment:
we will find out in the loop below whether files are readable. don't do it twice.